internal/kubernetes/nodefilters_test.go

```diff
@@ -104,6 +104,17 @@ func TestNodeLabelFilter(t *testing.T) {
 			},
 			passesFilter: true,
 		},
+		{
+			name: "FilterWithEmptyValue",
+			obj: &core.Node{
+				ObjectMeta: meta.ObjectMeta{
+					Name:   nodeName,
+					Labels: map[string]string{"cool": "very"},
```

Comment:
I'd imagine that `meta.ObjectMeta` should have the label `keyWithNoValue`.

Reply:
not really: I wanted to test that the filter does reject that node. It is the filter that has an empty value, not the node it self.
What is maybe not clear is that there is no test that should a node with empty value being selected by a selector also with simple empty value. I am adding that test.

Reply:
@jacobstr I have added one more test to remove any ambiguity on filtering in/out cases.